src/core/execute.h

```diff
@@ -177,6 +177,14 @@ typedef enum ExecCleanMask {
         _EXEC_CLEAN_MASK_INVALID = -EINVAL,
 } ExecCleanMask;
 
+typedef enum PrivateTmp {
+        PRIVATE_TMP_DISABLED,
```

Comment:
Hmm, "disabled" is not usually a term we use for things that are kinda booleans, but also not. maybe off?